WooCommerce/Classes/ViewRelated/Dashboard/Blaze/BlazeCampaignDashboardView.swift

```diff
@@ -40,17 +54,59 @@ final class BlazeCampaignDashboardViewHostingController: SelfSizingHostingContro
 }
 
 private extension BlazeCampaignDashboardViewHostingController {
-    /// Handles navigation to the campaign creation web view
+    private func createBlazeNavigationController() {
+        blazeNavigationController = UINavigationController(rootViewController: createProductSelectorViewController())
+    }
+
+    private func createProductSelectorViewController() -> ProductSelectorViewController {
+        let productSelectorViewModel = ProductSelectorViewModel(
+            siteID: viewModel.siteID,
+            purchasableItemsOnly: true,
+            onProductSelectionStateChanged: { [weak self] product in
+                guard let self = self else { return }
+
+                // Navigate to Campaign Creation Form once a simple/bundle product is selected.
```

Comment:
Nit: Should the comment mention products regardless of type because we treat every type as simple?

Reply:
Oh, right. Good find. This was a comment that I forgot to change when removing the `onVariationSelectionStateChanged` part. Fixed in https://github.com/woocommerce/woocommerce-ios/pull/11625/commits/c001d8600275c9e458a6f0387b2949595110612c